cmd/config.go

```diff
@@ -64,14 +64,16 @@ func initConfig(rootCmd *cobra.Command, rootOptions *options.RootOptions) error
 					if len(configValue) > 0 {
 						configValueStr := strings.Join(configValue, ",")
 						if err := flags.Set(f.Name, configValueStr); err != nil {
-							log.Errorf("failed to set config value: %s", err)
+							err := fmt.Errorf("failed to set config value: %w", err)
+							log.Error(err)
```

Comment:
Same question as the other PR, any reason to not have this as a single line with `log.Errorf()`

Reply:
Hey 👋 - I answered in the other PR: https://github.com/in-toto/go-witness/pull/83#discussion_r1410521223

Reply:
sorry if this was all very unclear. The reason for the splitting out the error formatting to `fmt.Errorf` is because `log.Errorf` does not support the `%w` directive. The reason for not having it all on a single line (e.g., `log.Errorf(fmt.Errorf(...` was for ease of readability.

After being round the houses with all of this a little bit, I believe we can switch all this to a `log.WithError(err).Error(...`. I will push said changes to this PR 😄